modules/extra-data.yaml

```diff
@@ -0,0 +1,13 @@
+name: extra-data
+buildsystem: simple
+sources:
+  - type: dir
+    path: data
+    dest: data
+build-commands:
+  - install -Dm644 data/com.spacestation14.Launcher.desktop -t /app/share/applications
+  - install -Dm644 data/com.spacestation14.Launcher.appdata.xml -t /app/share/appdata
+  - install -Dm644 -T data/icons/icon-64x64.png /app/share/icons/hicolor/64x64/apps/com.spacestation14.Launcher.png
+  - install -Dm644 -T data/icons/icon-128x128.png /app/share/icons/hicolor/128x128/apps/com.spacestation14.Launcher.png
+  - install -Dm644 -T data/icons/icon-256x256.png /app/share/icons/hicolor/256x256/apps/com.spacestation14.Launcher.png
+  - install -Dm644 -T data/icons/icon-scalable.svg /app/share/icons/hicolor/scalable/apps/com.spacestation14.Launcher.svg
```

Comment:
Since uou have scalable SVG and the icon don't come from the upstream package. then only the SVG should be here.